app/controllers/students_controller.rb

```diff
@@ -0,0 +1,11 @@
+class StudentsController < ApplicationController
+
+  def index
+    @students = Student.all
+  end
+
+  def show
+    @students = Student.find(:id)
```

Comment:
Is `@students` a good variable name for something that only contains one student?